sonic-net/sonic-buildimage
# [Marvell] Update mrvl-prestera SDK driver

#### Why I did it
Update mrvl-prestera SDK driver with below fixes,

* [mvSai] Add netdev notifier call - Change adds netdev notifier call to sai module to handle cleanup in case of unregister event in netdev.
* [ethDriver] Populate TX_DSA Tag(isTagged and VID) from egress packet - Change fixes the update of SAI_ROUTER_INTERFACE_STAT_OUT_OCTETS.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

#### How to verify it
Verified by running following test, 
1) config reload
2) syncd process kill
3) Check SAI_ROUTER_INTERFACE_STAT_OUT_OCTETS counter updates.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
